### ISSUE TITLE
1.20.0-beta0 unable to load on Windows

### ISSUE THREAD

**Issue by Positive07** [atom/atom#15194]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

Note that clearing the state would fix this issue, but the issue should still be fixed in some other way.

### Description

Can't open projects from the previous version of Atom Beta

### Steps to Reproduce

1. Install 1.19 beta 7
2. Open a project, some tabs, and the About page (maybe the Settings View too)
3. Update to 1.20 beta 0 automatically (Squirrel should download the update and you need to Restart and Install once it's downloaded)
4. Reopen Atom once the update completes (the error should happen automatically)
5. If the error doesn't appear, open a project from `File > Reopen Project`

Note that if you manually add a project with `Add Project Folder` or open a project you had already opened in 1.20 the issue doesn't appear

**Expected behavior:** Projects should open in tree view and tabs should be restored

**Actual behavior:** Instead, the error pop-ups and no editor is opened

**Reproduces how often:** Always

### Versions

This happened when updating to 1.20.0 beta0 on Windows 10 x64

```
> apm --version

apm  1.18.3
npm  3.10.10
node 6.9.5 x64
python 2.7.13
git 2.13.0.windows.1
visual studio 2015
```

### Additional Information
Stack trace:
```
Uncaught (in promise) TypeError: Cannot read property 'node_modules\about\styles\about.less' of undefined
    at ThemeManager.module.exports.ThemeManager.loadLessStylesheet (C:\Users\nano0\AppData\Local\atom\app-1.20.0-beta0\resources\app\src\theme-manager.js:294:65)
    at ThemeManager.module.exports.ThemeManager.loadStylesheet (C:\Users\nano0\AppData\Local\atom\app-1.20.0-beta0\resources\app\src\theme-manager.js:271:27)
    at C:\Users\nano0\AppData\Local\atom\app-1.20.0-beta0\resources\app\src\package.js:545:60
    at Array.map (native)
    at Package.module.exports.Package.loadStylesheets (C:\Users\nano0\AppData\Local\atom\app-1.20.0-beta0\resources\app\src\package.js:543:65)
    at C:\Users\nano0\AppData\Local\atom\app-1.20.0-beta0\resources\app\src\package.js:137:23
    at Package.module.exports.Package.measure (C:\Users\nano0\AppData\Local\atom\app-1.20.0-beta0\resources\app\src\package.js:99:21)
    at Package.module.exports.Package.finishLoading (C:\Users\nano0\AppData\Local\atom\app-1.20.0-beta0\resources\app\src\package.js:133:25)
    at PackageManager.module.exports.PackageManager.loadAvailablePackage (C:\Users\nano0\AppData\Local\atom\app-1.20.0-beta0\resources\app\src\package-manager.js:521:38)
    at C:\Users\nano0\AppData\Local\atom\app-1.20.0-beta0\resources\app\src\package-manager.js:479:25
    at Config.module.exports.Config.transact (C:\Users\nano0\AppData\Local\atom\app-1.20.0-beta0\resources\app\src\config.js:321:22)
    at PackageManager.module.exports.PackageManager.loadPackages (C:\Users\nano0\AppData\Local\atom\app-1.20.0-beta0\resources\app\src\package-manager.js:473:25)
    at C:\Users\nano0\AppData\Local\atom\app-1.20.0-beta0\resources\app\src\atom-environment.js:810:34
```

My settings can be found on this [Gist](https://gist.github.com/Positive07/623ef49857e10c67eabc8ba93f68ae22)

**Comment by Positive07** [comment]
Some other notes, after having this error, I can't Reopen any other project, nor Add Project Folder since the TreeView doesn't exist...

I had some other problems with 1.20 like the disappearance of the `apm` and `atom` commands from the command line (the `PATH` variable wasn't correctly updated when Squirrel installed the new 1.20 update). So once I have helped debugging this I'll go to 1.19 stable

**Comment by Stanzilla** [comment]
Exactly the same here, but since `Restart and update`never works for me (https://github.com/atom/atom/issues/6612) I waited a few seconds after Atom closed and then manually started it with having the option to restore with my usual project enabled.

**Comment by rwatts3** [comment]
Same issue just upgrade and can't open projects now.

**Comment by Arcanemagus** [comment]
If it helps track things down, running `atom --clear-window-state` **doesn't** help with this issue, but I might be seeing something slightly different as I am unable to open anything, even "projects" that Atom hasn't seen before.

**Comment by ungb** [comment]
I'm able to repro the issue on windows and am not able to open anything.  Following up with team now on this issue.

**Comment by Positive07** [comment]
This turned out to be a complete break for me, I had to use Open in Dev Mode to open projects, I'll roll to 1.19 stable because I cannot wait for the fix.

**Comment by pingshunhuangalex** [comment]
Just to add to the pool. I got completely blocked out from Atom. All I get is the blank screen as shown below. I've tried running `atom --safe`, `atom --clear-window-state`, `delete storage directory`, `uninstall and reinstall from standalone installer`, none of it works.

![image](https://user-images.githubusercontent.com/17170199/29100449-ee870224-7cee-11e7-9176-9adddf6c01a7.png)

**Comment by smashwilson** [comment]
I can replicate just by downloading and launching the current Atom Beta, but rebuilding from the tip of `1.20-releases` seems to be working. It might be a broken installer?

**Comment by smashwilson** [comment]
In the console, you can see that `snapshotAuxiliaryData.lessSourcesByRelativeFilePath` is undefined. Tracing through the source, that should be being populated by this part of the build:

https://github.com/atom/atom/blob/1dd776f237e36d0fba95ad3ada3ec2b9117ac9c3/script/lib/prebuild-less-cache.js#L107-L109

That function [did run](https://ci.appveyor.com/project/Atom/atom/build/7528/job/kyvvdjqxv153tfj8#L553) as part of the build that produced the 1.20-beta installer, so I'm not sure why it isn't there.

The only key that _is_ present in `snapshotAuxiliaryData` is `snapshotSections`. On my successful install from source, `snapshotAuxiliaryData` has three keys: `importedFilePathsByRelativeImportPath`, `lessSourcesByRelativeFilePath`, and `snapshotSections`. So we're missing the auxiliary data from the build somehow.

**Comment by Positive07** [comment]
Could that be related to that faulty installer not updating the PATH variable correctly?

**Comment by 50Wliu** [comment]
No, PATH stuff is handled by Squirrel.  There's a separate issue for that at #12124.

**Comment by nihiven** [comment]
I was having the same issue. I use this as my daily driver so I downgraded to 1.19.0-beta7 by deleting the 1.20 directory from \AppData\Local\atom and \AppData\Local\atom\packages. I'm not sure if this is the correct downgrade procedure, but it worked for me.

**Comment by Arcanemagus** [comment]
@nihiven You should install v1.19.0 (stable) for now, not v1.19.0-beta7.

**Comment by nihiven** [comment]
@Arcanemagus Thanks, I hadn't thought of that.

**Comment by Methroix** [comment]
I'm experiencing the same issue using the built-in updater.

However, I got it working by (somwhat) following on the findings of @smashwilson.
Instead of building it myself I simply downloaded the zip-file for 1.20.0-beta0, and it ran without hiccups.

**Comment by hughlomas** [comment]
Same issue on Win 8.1 x64, blank screen with menu bar and nothing else.

As nihiven suggested, I deleted the 1.20 files from `\AppData\Local\atom` and `\AppData\Local\atom\packages`, which allowed the editor to load as normal.

**Comment by jasonrudolph** [comment]
We've published [1.20.0-beta1](https://github.com/atom/atom/releases/tag/v1.20.0-beta1) which should resolve this issue.

If you run into any problems auto-updating from 1.20.0-beta0 to 1.20.0-beta1, you may need to uninstall your current Atom Beta app and then install the [latest beta release](https://atom.io/beta). If you're still seeing this issue after installing 1.20.0-beta1, please let us know.
